rtd_requirements.txt

```diff
@@ -10,6 +10,7 @@ pygments==2.14.0
 pytest==7.2.1
 redis==4.5.1
 sphinx>=2
+sphinx_rtd_theme==1.2.0
```

Comment:
Is this something new to get the builds working?

Reply:
Yeah, the RTD builds were failing. Without this, the CI was installing a very old version of sphinx_rtd_theme, which was causing a crash. I saw this in a few of our other PRs as well.

Reply:
Awesome, thank you for catching that.